src/Shared/HashCodeCombiner/HashCodeCombiner.cs

```diff
@@ -25,25 +25,6 @@ private HashCodeCombiner(long seed)
             _combinedHash64 = seed;
         }
 
-        [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public void Add(IEnumerable e)
```

Comment:
We don't care about this?  Presumably it's just calling the generic version now?

Reply:
I didn't see any uses for this and it seemed messier to add a generic version. Regardless, most of our codebase is using `System.HashCode` and Razor is an outlier due to it targeting ns2.0.

Reply:
Did you see any uses of this _before_ the introduction of the generic form? Derived types would prefer to call the generic form than cast to `IEnumerable`.

Reply:
I just checked and I don't see any references prior to this PR.